### PR TITLE
Dev -> Master v2.5.1

### PR DESCRIPTION
**Pull request by mirpedrol**
## PR checklist

- [x] This comment contains a description of changes (with reason)
- [x] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
 # nf-core/tools: Changelog
 
+## v2.6dev
```
Comment (drpatelh) [inline CHANGELOG.md:3]: Need to update the `CHANGELOG`

**Review by ewels — APPROVED** [review]
LGTM!

**Review by mashehu — APPROVED** [review]

**Comment by mashehu** [comment]
Do we want to merge #1788 into this?

**Comment by mirpedrol** [comment]
> Do we want to merge #1788 into this?

Yes! I merged that. 
And I think we can merge this PR to solve the bug and move new issues to 2.6, do you agree?

**Pull request merged**